### DISCUSSION TITLE
Issue that i can't login in with demo. Failet to fetch.

### DISCUSSION THREAD

**Discussion by gitHuangweize** [OktopUSP/oktopus#270] [Q&A] [answered]
When i user the command start the controller, and i login in with demo, it show "**Failed to fetch**". How can i solve this? Help!
the command: **COMPOSE_PROFILES=nats,controller,cwmp,mqtt,stomp,ws,adapter,frontend,portainer docker compose up** 
![image](https://github.com/OktopUSP/oktopus/assets/68935666/735b6059-50da-4213-8c53-b32428421c4e)

**Comment by leandrofars** [comment]
all the containers are running, any container have died?

**Reply by gitHuangweize** [reply]
![image](https://github.com/OktopUSP/oktopus/assets/68935666/60f91a17-7eee-4cb9-8c15-e3c3f1a450f3)
There are 14 container running. No any containers have died.
But i found some error when starting these contariners.  like "Error to load environment variables: open .env: no such file or directory".

**Comment by gitHuangweize** [comment, marked as answer]
This issue occurred because I used a public IP to access it, which led to a cross-domain problem, go to the deploy/compose directory and configure the ".env.frontend" file to your public IP.
![image](https://github.com/OktopUSP/oktopus/assets/68935666/6a69ee9c-501f-4fdd-b3bf-df7a6b378054)

![image](https://github.com/OktopUSP/oktopus/assets/68935666/df87ebdd-c0e3-4dc4-8f5e-d9302e8d8cb1)

**Reply by leandrofars** [reply]
Great, I'll add it to the README file, many people are having the same problem.
